compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -422,7 +422,7 @@ getVectorDistributeReductionConfig(
     ArrayAttr threadBasisAttr = b.getArrayAttr(
         {b.getI64ArrayAttr(threadCounts), b.getI64ArrayAttr(mapping)});
 
-    SmallVector<NamedAttribute, 4> configAttrs = {
+    SmallVector<NamedAttribute, 5> configAttrs = {
```

Comment:
You can also make this a plain array with the size deduced
```suggestion
    NamedAttribute configAttrs[] = {
```